src/test/java/api/loans/RenewalAPITests.java

```diff
@@ -769,6 +771,36 @@ public void cannotRenewWhenItemIsNotLoanable() {
       hasLoanPolicyNameParameter("Non loanable policy"))));
   }
 
+  @Test
+  public void cannotRenewWhenItemIsDeclaredLost() {
+
+    final IndividualResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
+    final IndividualResource jessica = usersFixture.jessica();
+
+    final JsonObject loanJson = loansFixture.checkOutByBarcode(smallAngryPlanet, usersFixture.jessica())
+      .getJson();
+
+    declareItemLost(loanJson);
+
+    final Response response = attemptRenewal(smallAngryPlanet, jessica);
+
+    assertThat(response.getJson(), hasErrorWith(allOf(
+      hasMessage("item cannot be renewed: item is Declared lost"),
```

Comment:
Has this error message been checked with the product owner, it seems differently formatted to the other messages?

Reply:
The message is according to story's requirement.  I am waiting the confirmation of  product owner.

Reply:
Thank you